exonum/src/events/codec.rs

```diff
@@ -37,11 +44,11 @@ impl Decoder for MessagesCodec {
         }
         // Check payload len
         let total_len = LittleEndian::read_u32(&buf[6..10]) as usize;
-        if total_len > MAX_MESSAGE_LEN {
+        if total_len as u32 > self.max_message_len {
```

Comment:
remove `as usize`  and `as u32`

Reply:
@vldm, there are two places where we have to cast it to `usize` again. See the code below `// Read message` comment. I think it doesn't matter here.